.github/workflows/lint-build-test.yaml

```diff
@@ -0,0 +1,16 @@
+name: Lint, build and test
+
+on:
+  push:
+    branches:
+      - stage
```

Comment:
shouldn't master be added here as well? The base branch can change again after the PR has been made 🤔

Reply:
Yea I can also add `master` here. But if you look at the line below, you can see `pull_request`. That means that on every PR for every base this workflow will run. In addition, we run on pushing / merging to `stage` via this `push` trigger.

Reply:
Ah ok yea then I will add master here. makes sense